### ISSUE TITLE
Getting a 404 when hitting /good-beers

### ISSUE THREAD

**Issue by brocktubre** [stormpath/stormpath-spring-boot-angular-pwa-example#3]
I am at the point where I am hitting the end point http://localhost/good-beers and prompted with username and password. After entering in creditials, a 404 page not found is being displayed. Here is what httpie returns. 

```
HTTP/1.1 404 
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Content-Type: application/hal+json;charset=UTF-8
Date: Wed, 07 Jun 2017 14:23:14 GMT
Expires: 0
Pragma: no-cache
Set-Cookie: JSESSIONID=032570761EDD421A0B5B10F187A8DA73; Path=/; HttpOnly
Transfer-Encoding: chunked
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-XSS-Protection: 1; mode=block

{
    "error": "Not Found",
    "message": "No message available",
    "path": "/good-beers",
    "status": 404,
    "timestamp": 1496845394241
}
```

I should be returned a list of the beers. Thanks in advance!

**Comment by mraible** [comment]
Hello @brocktubre. Do you already have a Stormpath account? If so, things should work. If not, I have a tutorial that uses Okta (because [Stormpath joined forces with Okta](https://stormpath.com/blog/stormpaths-new-path?utm_source=github&utm_medium=readme&utm-campaign=okta-announcement) back in February). 

https://github.com/oktadeveloper/okta-spring-boot-angular-pwa-example/blob/master/TUTORIAL.md

Please visit [the Migration FAQs](https://stormpath.com/oktaplusstormpath?utm_source=github&utm_medium=readme&utm-campaign=okta-announcement) for a detailed look at what this means for Stormpath users.

**Comment by brocktubre** [comment]
Yes, I already have a Strompath account and the user is registered within a new application I created.  Since the app is brand new, I would like to use the most up to date tutorial (Stormpath/Okta user authentication).

**Comment by mraible** [comment]
If you're planning on migrate to Okta, I'd suggest using the Okta version I posted above. It shows how to use the Stormpath SDKs with Okta, so the code is very similar.
